src/App/Pages/Vault/GroupingsPage/GroupingsPageViewModel.cs

```diff
@@ -244,6 +247,18 @@ public async Task LoadAsync()
                     groupedItems.Add(new GroupingsPageListGroup(noFolderCiphersListItems, AppResources.FolderNone,
                         noFolderCiphersListItems.Count, uppercaseGroupNames, false));
                 }
+                // Ensure this is last in the list (appears at the bottom)
```

Comment:
This can be potentially difficult to get to based on the size if the vault.  My personal preference would be for this to appear at the end of the `Types` section.  Not sure how others feel though.

Reply:
Agreed, but trash I would consider low-priority/not often used so may be okay.. this is the same place it's located in the browser extension as well (bottom)

Reply:
Understood.  PR looks legit.